orcid-core/src/main/java/org/orcid/pojo/Spam.java

```diff
@@ -0,0 +1,79 @@
+package org.orcid.pojo;
+
+import java.io.Serializable;
+
+import org.orcid.pojo.ajaxForm.Date;
+
+public class Spam implements Serializable {
+    private static final long serialVersionUID = 1L;
+    
+    private Date createdDate;
+    
+    private Date lastModifiedDate;
+
+    private Date reportedDate;
+
+    private String sourceType;
+
+    private Integer count;
+    
+    public Date getCreatedDate() {
+        return createdDate;
+    }
+    
+    public void setCreatedDate(Date createdDate) {
+        this.createdDate = createdDate;
+    }
+
+    public void setLastModifiedDate(Date lastModifiedDate) {
+        this.lastModifiedDate = lastModifiedDate;
+    }
+    
+    public Date getLastModifiedDate() {
+        return lastModifiedDate;
+    }
+
+    public Date getReportedDate() {
+        return reportedDate;
+    }
+
+    public void setReportedDate(Date reportedDate) {
+        this.reportedDate = reportedDate;
+    }
+
+    public String getSourceType() {
+        return sourceType;
+    }
+
+    public void setSourceType(String sourceType) {
+        this.sourceType = sourceType;
+    }
+
+    public Integer getCount() {
+        return count;
+    }
+
+    public void setCount(Integer count) {
+        this.count = count;
+    }
+    
+    public static Spam fromValue(org.orcid.jaxb.model.v3.release.record.Spam spam) {
+        if (spam == null)
+            return null;
+        
+        Spam s = new Spam();
```

Comment:
@DanielPalafox please remove the extra spaces and apply the ORCID formatting rules.